code/modules/modular_computers/computers/machinery/console_presets.dm

```diff
@@ -19,96 +20,77 @@
-
+	 _app_preset_name = "engineering"
+	 _enrolled = 1
 
 // ===== MEDICAL CONSOLE =====
 /obj/machinery/modular_computer/console/preset/medical
 	 console_department = "Medbay"
 	 desc = "A stationary computer. This one comes preloaded with medical programs."
-	 _software_locked = 1
-
-/obj/machinery/modular_computer/console/preset/medical/install_programs()
-	cpu.hard_drive.store_file(new/datum/computer_file/program/chatclient())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/suit_sensors())
-
+	 _app_preset_name = "medical"
+	 _enrolled = 1
 
 // ===== RESEARCH CONSOLE =====
 /obj/machinery/modular_computer/console/preset/research
 	 console_department = "Research"
 	 desc = "A stationary computer. This one comes preloaded with research programs."
+	 _app_preset_name = "research"
 	 _has_aislot = 1
-	 _software_locked = 1
-
-/obj/machinery/modular_computer/console/preset/research/install_programs()
-	cpu.hard_drive.store_file(new/datum/computer_file/program/ntnetmonitor())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/chatclient())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/aidiag())
-
+	 _enrolled = 1
 
 // ===== COMMAND CONSOLE =====
 /obj/machinery/modular_computer/console/preset/command
 	 console_department = "Command"
 	 desc = "A stationary computer. This one comes preloaded with command programs."
+	 _app_preset_name = "command"
 	 _has_id_slot = 1
 	 _has_printer = 1
-	 _software_locked = 1
-
-/obj/machinery/modular_computer/console/preset/command/install_programs()
-	cpu.hard_drive.store_file(new/datum/computer_file/program/chatclient())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/card_mod())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/comm())
-
-/obj/machinery/modular_computer/console/preset/command/main
-	 console_department = "Command"
-	 desc = "A stationary computer. This one comes preloaded with essential command programs."
+	 _enrolled = 1
 
 // ===== SECURITY CONSOLE =====
 /obj/machinery/modular_computer/console/preset/security
 	 console_department = "Security"
 	 desc = "A stationary computer. This one comes preloaded with security programs."
-	 _software_locked = 1
-
-/obj/machinery/modular_computer/console/preset/security/install_programs()
-	cpu.hard_drive.store_file(new/datum/computer_file/program/chatclient())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/camera_monitor())
+	 _app_preset_name = "security"
+	 _enrolled = 1
 
 // ===== CIVILIAN CONSOLE =====
 /obj/machinery/modular_computer/console/preset/civilian
 	 console_department = "Civilian"
 	 desc = "A stationary computer. This one comes preloaded with generic programs."
-
-/obj/machinery/modular_computer/console/preset/civilian/install_programs()
-	cpu.hard_drive.store_file(new/datum/computer_file/program/chatclient())
-	cpu.hard_drive.store_file(new/datum/computer_file/program/nttransfer())
+	 _app_preset_name = "civilian"
+	 _enrolled = 1
```

Comment:
Yes they are restricted as is every other console.
If you want to be able to download apps, ... Get a laptop or engineering to build you a console and then enroll it as a private device